redi-connect-front/src/components/organismms/LoggedOutNavbar.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from "react";
+import { Navbar } from "react-bulma-components";
+import Button from "../atoms/Button";
+import Logo from "../atoms/Logo";
+
+const LoggedOutNavbar = ({ children }: any) => {
+  const [active, setActive] = useState(false);
+
+  return (
+    <Navbar active={active}>
+      <div className="container">
+        <Navbar.Brand>
+          <Navbar.Item>
+            <Logo />
+          </Navbar.Item>
+          <Navbar.Burger
```

Comment:
Nice :) But the burger menu is activated at already < 1024px, which seems very high. Consider reducing it?

Reply:
The Navbar and it's behaviour is just placeholder for now. Once we set the container width and breakpoints within Bulma and agreed on weather to have one Navbar with a prop to distinguish between logged in or logged out vs. two seperate Navbar components, we will completely redo the Navbar. I also put it in the description as todo.

Reply:
Sounds great!